.github/workflows/integration-tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         if: matrix.platform == 'ubuntu-latest'
```

Comment:
This should be fixed to `22.04` as it will break when github updates to the new LTS `24.04` as webkit2gtk 4.0 is no longer available in their repositories.

Reply:
we should do that for the entire tauri 1.x and plugin v1 branches